api/client/client.go

```diff
@@ -1632,7 +1632,7 @@ func (c *Client) SearchEvents(ctx context.Context, fromUTC, toUTC time.Time, nam
 }
 
 // SearchSessionEvents allows searching for session events with a full pagination support.
-func (c *Client) SearchSessionEvents(ctx context.Context, fromUTC time.Time, toUTC time.Time, limit int, order types.EventOrder, startKey string) ([]events.AuditEvent, string, error) {
+func (c *Client) SearchSessionEvents(ctx context.Context, fromUTC, toUTC time.Time, limit int, order types.EventOrder, startKey string, cond *types.WhereExpr) ([]events.AuditEvent, string, error) {
```

Comment:
You may notice the new parameter `cond` isn't actually used in the function body. The reason the parameter is added here is just to satisfy the `events.IAuditLog` interface that needs to be implemented by both server and client. Therefore it might be misleading to include full documentation here.

I actually reverted this `api/client` change and left the `cond` "glue" parameter only in `lib/auth/clt.go` that suffices to satisfy the interface.